camp/nfutsum/nfutsum-miniproject/XBox Controller Left and Right bumper/src/main/java/frc/robot/Robot.java

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
+/**
+ * The VM is configured to automatically run this class, and to call the functions corresponding to
+ * each mode, as described in the TimedRobot documentation. If you change the name of this class or
+ * the package after creating this project, you must also update the build.gradle file in the
+ * project.
+ */
+public class Robot extends TimedRobot {
+  private Command m_autonomousCommand;
+
+private RobotContainer m_robotContainer;
```

Comment:
So I know that the default wpilib project creates variables with m_ so its ok for now, but code you create / we checkin in season should not do this.

Why, you might ask?  It sets a bar that others should follow.  Doing this reduces confusion.  Readers won't be induced to ask ... "Why do they name it 'm_', is it special?  Well I better read it more carefully."  So that the logic folks see and read is neat and orderly.  So the code is consistant and inspires confidence.